packages/core/test/foundry/optimistic-assertor/OptimisticAsserter.EdgeCases.t.sol

```diff
@@ -25,11 +15,11 @@ contract InvalidParameters is Test {
         defaultCurrency.approve(address(optimisticAssertor), optimisticAssertor.defaultBond() * 2);
 
         // Account1 asserts a claim.
-        bytes32 assertionId = optimisticAssertor.assertTruth(bytes(claimAssertion));
+        bytes32 assertionId = optimisticAssertor.assertTruth(trueClaimAssertion);
```

Comment:
```suggestion
        bytes32 assertionId = optimisticAssertor.assertTruth(falseClaimAssertion);
```

better use `falseClaimAssertion` from common as this test double dispute, so most likely should be false claim